tests/integration/test_audio_video_services_batch4.py

```
@@ -1657,21 +1655,20 @@ def test_process_file_returns_processed_file(self, tmp_path: Path) -> None:
         mock_model.generate = MagicMock(return_value="programming")
 
-        with patch("services.text_processor.ensure_nltk_data"):
+        with patch(
+            "services.text_processor.read_file",
+            return_value="Python programming content",
+        ):
             with patch(
-                "services.text_processor.read_file",
+                "services.text_processor.truncate_text",
                 return_value="Python programming content",
             ):
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)